projects/angular/src/data/datagrid/_properties.datagrid.scss

```
@@ -9,86 +9,78 @@
+    --clr-datagrid-pagination-btn-disabled-color: #{tokens.$cds-alias-status-disabled};
+    --clr-datagrid-pagination-input-border-color: #{tokens.$cds-alias-object-border-color};
+    --clr-datagrid-pagination-input-border-focus-color: var(--clr-color-action-400);
+    --clr-datagrid-popover-bg-color: #{tokens.$cds-alias-object-container-background};
+    --clr-datagrid-popover-border-color: #{tokens.$cds-alias-object-border-color};
+    --clr-datagrid-popover-font-color: #{tokens.$cds-alias-typography-color-400};
+    --clr-datagrid-action-popover-hover-color: #{tokens.$cds-alias-object-interaction-background-hover};
+    --clr-datagrid-row-selected: #{tokens.$cds-alias-typography-color-500};
+    --clr-datagrid-row-selected-background-color: #{tokens.$cds-alias-object-interaction-background-selected};
 
-      --clr-datagrid-row-hover-color: var(--clr-color-neutral-200);
-      --clr-datagrid-row-hover-font-color: var(--clr-color-neutral-1000);
-      --clr-datagrid-action-toggle-color: var(--clr-color-neutral-700);
-      --clr-datagrid-pagination-btn-color: var(--clr-color-neutral-700);
-      --clr-datagrid-pagination-btn-disabled-color: var(--clr-color-neutral-600);
-      --clr-datagrid-pagination-input-border-color: var(--clr-datagrid-default-border-color);
-      --clr-datagrid-pagination-input-border-focus-color: var(--clr-color-action-400);
-      --clr-datagrid-popover-bg-color: var(--clr-color-neutral-0);
-      --clr-datagrid-popover-padding: #{variables.$clr_baselineRem_0_25};
-      --clr-datagrid-popover-border-color: var(--clr-datagrid-default-border-color);
-      --clr-datagrid-action-popover-hover-color: var(--clr-color-neutral-200);
-      --clr-datagrid-row-selected: var(--clr-color-neutral-1000);
-      --clr-datagrid-loading-background: #{datagrid-variables.$clr-datagrid-loading-background};
-      --clr-datagrid-popovers-box-shadow-color: rgba(140, 140, 140, 0.25);
-      --clr-datagrid-popovers-box-shadow: 0
-        #{variables.$clr_baselineRem_1px}
-        #{variables.$clr_baselineRem_3px}
-        var(--clr-datagrid-popovers-box-shadow-color);
-      --clr-datagrid-popover-font-color: #{dropdown-variables.$clr-dropdown-text-color};
+    --clr-datagrid-column-switch-header-font-color: #{tokens.$cds-alias-object-interaction-color};
+    --clr-datagrid-column-switch-header-font-hover-color: #{tokens.$cds-alias-object-interaction-color-hover};
+    --clr-datagrid-column-switch-header-font-hover-active: #{tokens.$cds-alias-object-interaction-color-active};
 
-      --clr-datagrid-column-switch-header-font-color: var(--clr-color-neutral-500);
-      --clr-datagrid-column-switch-header-font-hover-color: var(--clr-color-action-600);
+    --clr-datagrid-detail-caret-icon-open-bg-color: #{tokens.$cds-alias-object-interaction-background-highlight};
+    --clr-datagrid-detail-caret-icon-open-icon-color: #{tokens.$cds-global-color-gray-0};
```

Spec wise this should be `cds-alias-object-interaction-color`.

```suggestion
    --clr-datagrid-detail-caret-icon-open-icon-color: #{tokens.$cds-alias-object-interaction-color};
```